test/builtin/test_assignment.py

```diff
@@ -276,7 +268,7 @@ def test_set_and_clear(str_expr, str_expected, msg):
         (None, None, None),
         (
             (
-                "A[x_]:=B[x];B[x_]:=F[x_];F[x_]:=G[x];"
+                "A[x_]:=B[x];B[x_]:=F[x];F[x_]:=G[x];"
```

Comment:
In this test, `SetDelayed` or `:-=` is getting used. That is at a level _below_  the generic `mathics.builtin`. Either rewrite this to avoid using `SetDelayed` and the built-in functions, or move it under `test/builtin/assignments`. Thanks.

Reply:
In this case, maybe is the same to use `=` and `:=`. But you mean to use the FullForm of the expressions?

Reply:
> In this case, maybe is the same to use `=` and `:=`. But you mean to use the FullForm of the expressions?

No. I guess I am not being clear here. I was using this as an example of the entire suite of tests that seems to be testing the `Set*` functions which appear in module `mathics.builtin.assignments`, not `mathics.builtin`.

But let me back up. What is the intention of this suite of tests?

As a principle, code in `test/builtin` should not use any code from the builtin modules under such as any of the `Set` functions.

Reply:
Yes, it does. Thanks!